backend/app/assets/stylesheets/spree/backend/components/_number_with_currency.scss

```diff
@@ -0,0 +1,34 @@
+.number-with-currency {
+  &-symbol {
+    min-width: 2.5em;
+    text-align: center;
+  }
+
+  &-amount {
+    text-align: right;
+  }
+
+  &-addon {
+    width: 5.5rem;
+    text-align: left;
+  }
+
+  & &-select {
+    padding: 0;
+    background: #fff;
+
+    select {
+      @extend .c-select;
+
+      display: inline-block;
+      width: 100%;
+      height: 31px;
```

Comment:
This leaves a `1px` gap on the top and bottom (height is `2.25rem == 29px`), but I don't think that's a huge issue, and is worth it to avoid this magic number and ensure the input will scale well if our font size changes.